doc/content/guides/connecting-gateways/nasysoutdoorgateway/_index.md

```diff
@@ -0,0 +1,184 @@
+---
+title: "NASYS LoRaWAN® Outdoor Gateway"
+description: ""
+weight: 1
+---
+
+Technical information regarding the LoRaWAN® Outdoor Gateway can be found in [the official product page][1]. This page guides you to connect it to {{% tts %}}.
+
+## Prerequisites
+
+1. User account on {{% tts %}} with rights to create Gateways.
+2. LoRaWAN® Outdoor Gateway connected to the Internet (or your local network) via Ethernet.
+
+## Registration
+
+Create a gateway by following the instructions for the [Console]({{< ref "/guides/getting-started/console#create-gateway" >}}) or the [CLI]({{< ref "/guides/getting-started/cli#create-gateway" >}}). Typically, the **EUI** field for your gateway should exist on the sticker at the bottom.
+
+## Configuration using a Terminal
+
+Find the IP address the gateway. This can be done in various ways. You can connect your machine to the same local network as that of the gateway Ethernet connection and scan for open SSH ports or assign a static IP to the gateway and use that. Once the gateway IP address is found, ssh into it.
+
+```bash
+$ ssh root@<GatewayIP>
+```
+
+The default username is **root**, and the default password can be also found in the sticker. It is typically suggested that you change the default password using `passwd` after logging in.
+
+Your gateway should come with a slightly modified version of the [Lora-net UDP packet forwarder](https://github.com/Lora-net/packet_forwarder) pre-installed at `/opt/nas-lgw`. There are two configuration files `global_conf.json` and `local_conf.json`, both located in `/opt/nas_lgw`.
+
+Typically, you should not need to touch the `global_conf.json` file. Instead, edit `local_conf.json`
+
+```bash
+$ cp /opt/nas-lgw/local_conf.json /opt/nas-lgw/local_conf.json.bak
+$ vi /opt/nas-lgw/local_conf.json
+```
+
+Edit the server parameters.
```

Comment:
With v3 we can automatically configure gateways with not only server parameters, but also with its frequency plan:

```
$ curl -H "Authorization: Bearer NNSXS.<snip>.<snip>" https://thethings.example.com/api/v3/gcs/gateways/your-gateway-id/semtechudp/global_conf.json
```

@rvolosatovs wrote a nice script for Kerlink Wirnet gateways, maybe we can do the same for the NASYS?

Reply:
Yes, please see https://github.com/TheThingsNetwork/kerlink-station-firmware/pull/9

Reply:
I can checkout the script and potentially try to make it work for the NASys gateway as well

Reply:
It appers that the nas-fwd `global_conf.json` and `local_conf.json` files use some vendor-specific configuration, so the `semtechudp/global_conf.json` endpoint is not enough. This makes it hard to use the existing provisioning script.

If we want to use the GCS, having the script update the gateway configuration in-place (so as to preserve vendor-specific extensions) instead of overwrite it could be a possible solution?

Reply:
Ok, if they use different configuration, we can just merge an initial version of this doc that doesn't use such a script.

Reply:
Afaik we should be adding an endpoint to GCS in that case, like we did for Kerlink.

Or aren't the vendor-specific settings known to GCS?

Reply:
After testing the GCS config works for NASys (at least with `EU_863_870`), so probably no new endpoints will be needed.

Reply:
Great